components/reusables/PackagesEmptyState.tsx

```diff
@@ -4,16 +4,26 @@ import { PackageIcon } from 'lucide-react';
 
 
 
-export const PackagesEmptyState  = ({searchTerm}: {searchTerm?: string}) => (
-        <div className="col-span-full flex flex-col items-center justify-center p-12 bg-gray-50 rounded-lg border-2 border-dashed border-gray-200">
-          <div className="w-16 h-16 bg-gray-100 rounded-full flex items-center justify-center mb-4">
-            <PackageIcon className="w-8 h-8 text-gray-400" />
-          </div>
-          <h3 className="text-lg font-semibold text-gray-700 mb-1">No Packages Found</h3>
-          <p className="text-sm text-gray-500 text-center max-w-sm">
-            {searchTerm
-              ? "No packages match your search criteria. Try different keywords."
-              : "There are currently no packages available for this category."}
-          </p>
-        </div>
-      );
+export const PackagesEmptyState = ({ searchTerm }: { searchTerm?: string }) => (
+  <div className="col-span-full flex flex-col items-center justify-center p-12 bg-gray-50 rounded-lg border-2 border-dashed border-gray-200">
+    <div className="w-16 h-16 bg-gray-100 rounded-full flex items-center justify-center mb-4">
+      <PackageIcon className="w-8 h-8 text-gray-400" />
+    </div>
+    <h3 className="text-lg font-semibold text-gray-700 mb-1">No Packages Found</h3>
+    <p className="text-sm text-gray-500 text-center max-w-sm">
+      {searchTerm
+        ? "No packages match your search criteria. Try different keywords."
+        : "There are currently no packages available for this category."}
+    </p>
+  </div>
+);
+
+export const TeamEmptyState  = ({searchTerm}: {searchTerm?: string}) => (
+  <div className="col-span-full flex flex-col items-center justify-center p-12 bg-gray-50 rounded-lg border-2 border-dashed border-gray-200">
+    <div className="w-16 h-16 bg-gray-100 rounded-full flex items-center justify-center mb-4">
+      <PackageIcon className="w-8 h-8 text-gray-400" />
```

Comment:
I think Team Empty state shouldn't be using Package icon we should find a different icon from FA that would represent it properly.